site/docs/build-ref.html

```diff
@@ -293,7 +293,7 @@ <h3 id="lexi">Lexical specification of a label</h3>
   The precise details of allowed target names are below.
 </p>
 
-<h4 id="name">Target names, <code>//...:<b>target-name</b></code></h4>
+<h4 id="name">Target names, <code>//xyz:<b>target-name</b></code></h4>
```

Comment:
I think the style is to use //<package name>:target-name or //path/to/package:target-name, instead of `xyz`.

Reply:
Fair enough. Works for me.
Done.